SetupScript.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+if [ $USER -ne "root"]
+	then
+	echo "Permission Denied"
+	exit 1
+fi
+#base script to fresh install systems onto base image for raspberry pi
+
+#check internet connection
+ping -q -c3 google.com > /dev/null
+
+if [ $? -ne 0 ]
+then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#configure displays
+cd /opt/
+git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git
+cd Epsilon-Raspberry/primary/
+cp xorg.conf /etc/X11/
+cp config.txt /boot/
+#install rabbit-mq
+
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | sudo tee /etc/apt/sources.list.d/rabbitmq.list && sudo apt-get update && sudo apt-get install rabbitmq-server
```

Comment:
If we're running as root, I don't think we need to have `sudo` under tee. Please verify